canonical/juju-dashboard-charm
# Upgrade dashboard to 0.11.2

## Changes:
- This handles the version endpoint more defensively so the dashboard continues to work when the API is not working.

# QA:
- Set up a Juju 3.2 controller.
- Follow the instructions for building the machine charm:
https://github.com/canonical/juju-dashboard-charm#building-and-testing-the-machine-charm
- Go to the controllers list and make sure you can see the controller.
- Turn off your wifi/internet and refresh the dashboard.
- The dashboard should display the controller and there shouldn't be any errors in your console about getting the version.
